packages/amplify-graphql-model-transformer/src/__tests__/model-transformer.test.ts

```diff
@@ -1682,4 +1682,33 @@ describe('ModelTransformer:', () => {
       });
     });
   });
+
+  it('should remove null timestamp fields from input', () => {
+    const validSchema = `
+      type Foo @model(timestamps: { updatedAt: null }) {
```

Comment:
Can we add the below test cases?
- Both values explicitly mentioned? (timestamps: { createdAt: "createdOn", updatedAt: "updatedOn" })
- Both values explicitly mentioned and one is set to null? (timestamps: { createdAt: "createdOn", updatedAt: null })